hw/top_earlgrey/dv/chip_rom_tests.hjson

```diff
@@ -41,6 +41,261 @@
         "+bypass_alert_ready_to_end_check=1",
       ]
     }
+    {
+      name: rom_e2e_sigverify_always_a_bad_b_bad_test_unlocked0
+      uvm_test_seq: chip_sw_rom_e2e_sigverify_always_a_bad_b_bad_vseq
+      sw_images: [
+        "//sw/device/silicon_creator/rom/e2e:empty_test_slot_a_corrupted:1:ot_flash_binary:signed:fake_test_key_0:no_sw_log_db",
```

Comment:
I wonder if we should let the `sw_logger_if` accept `sw_images` with no `elf` file, without bombing. That way, we will have one less `sw_image_flag` to deal with.

Reply:
I think you mean with no SW logging database files? This flag applies to the txt files that are loaded. (The ELF file loaded by the testbench is just used for the backdoor symbol overwriting, and it already has a [parameter](https://cs.opensource.google/opentitan/opentitan/+/master:hw/top_earlgrey/dv/env/seq_lib/chip_sw_base_vseq.sv;drc=7d93c75776da8020aadc42ecc74bd330a4aad73f;l=499) to avoid such a scenario.)

But yeah, I think that might be better, if there are no SW logging database text files, then just move along without  crashing the sim. I can add that cleanup in a followup PR too.

Reply:
Sorry, yes i meant the log database txt files. It is a very simple change here:
https://github.com/lowRISC/opentitan/blob/ac560a4e08106705a1d6730b96aa5f06504864a5/hw/dv/sv/sw_logger_if/sw_logger_if.sv#L157

Just replace the fatal log with info log.

Reply:
Fixed in #16366.